src/MorseL/HubWebSocketHandler.cs

```diff
@@ -49,7 +49,7 @@ public HubWebSocketHandler(IServiceProvider services, ILoggerFactory loggerFacto
             _logger = loggerFactory.CreateLogger<HubWebSocketHandler<THub, TClient>>();
             _serviceScopeFactory = services.GetRequiredService<IServiceScopeFactory>();
             _backplane = services.GetService<IBackplane>();
-            _morselOptions = services.GetService<IOptions<MorseLOptions>>().Value;
```

Comment:
I _think_ IOptions<...> is always treated as existing and thus IServiceProvider.GetService<IOptions<...>>() should always return a value. IOptions<...>.Value, on the other hand, may not. But that would throw a null reference exception in that case. TLDR; GetRequiredService may be superfluous but I'm not certain so let's keep it.

Reply:
I made this change in response to my mocked service collection not containing an `IOptions<MorseLOptions>` which would succeed (since it was only `GetService`) but then bomb out on the `.Value` call. I figured it was more ideal to change it to be a required service so that the failure is more explicit ("couldn't find required service" vs "a null reference occurred").

Reply:
That's probably because you manually added the dependency instead of using the Microsoft.Extensions.Options ```Configure<MorseLOptions>()``` configure extension method that actually adds the plumbing for all the options stuff.

Reply:
In normal operation - I think you'll still get a the null reference exception as I think the GetService will always return a value for IOptions - provided UseOptions has been called.

Reply:
Possibly. I'm still not sure how I feel about our prolific use of service location but assuming a "normally" configured app lifetime it probably isn't a huge deal.

What's your verdict? Keep this change or revert?

Reply:
Keep it. If anything, it will help catch errors in the tests.